setup.py

```diff
@@ -33,12 +33,13 @@
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.6'
     ],
     install_requires=[
+        'future',
         'futures',
         'threadloop>=1,<2',
-        # we want thrift>=0.9.2.post1,<0.9.3, but we let the users pin to that
-        'thrift',
+        'thrift>=0.10.0',
```

Comment:
It seems that generated files require thrift==0.10.0 to work correctly. With forcefully installed thrift==0.9.3, I'm receiving `ImportError: cannot import name 'TFrozenDict'` exception in `jaeger_client/thrift_gen/zipkincore/ZipkinCollector.py`, which makes sense: `TFrozenDict` appears only in 0.10.0 version.

Given this, I'm not sure how to go with Python 3 support:
- stubs generated by thrift 0.9 don't support Python 3
- stubs generated by thrift 0.10 do support Python 3, but they require thrift 0.10 package

Maybe there should be two sets of stubs generated (thrift_gen_0_10_0 and thrift_gen_0_9_3) and the code would switch between them with:
```
if sys.version_info[0] == 2  # or should it be a check for thrift version?:
    from jaeger_client import thrift_gen_0_9_3 as thrift_gen
  else:
    from jaeger_client import thrift_gen_0_10_0 as thrift_gen

import thrift_gen.zipkincore.ZipkinCollector as zipkin_collector
```

@yurishkuro what do you think?